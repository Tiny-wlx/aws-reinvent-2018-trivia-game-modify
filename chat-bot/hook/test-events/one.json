{
  "messageVersion": "1.0",
  "invocationSource": "DialogCodeHook",
  "userId": "ozw40htm8dvj0n1lk9ugt6gja961h4dl",
  "sessionAttributes": {
    "started": "true",
    "currentQuestion": "1",
    "currentScore": "0",
    "currentSlot": "one"
  },
  "requestAttributes": null,
  "bot": {
    "name": "TriviaGame",
    "alias": "$LATEST",
    "version": "$LATEST"
  },
  "outputDialogMode": "Text",
  "currentIntent": {
    "name": "LetsPlay",
    "slots": {
      "fifteen": null,
      "nine": null,
      "six": null,
      "one": 2012,
      "seven": null,
      "three": null,
      "two": null,
      "thirteen": null,
      "eight": null,
      "fourteen": null,
      "four": null,
      "sixteen": null,
      "twelve": null,
      "eleven": null,
      "ten": null,
      "five": null
    },
    "confirmationStatus": "None"
  },
  "inputTranscript": "Let's play"
}
