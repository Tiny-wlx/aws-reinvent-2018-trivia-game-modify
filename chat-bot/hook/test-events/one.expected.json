{
  "sessionAttributes": {
    "started": "true",
    "currentQuestion": 2,
    "currentScore": 100,
    "currentSlot": "two"
  },
  "dialogAction": {
    "type": "ElicitSlot",
    "intentName": "LetsPlay",
    "slots": {
      "fifteen": null,
      "nine": null,
      "six": null,
      "one": 2012,
      "seven": null,
      "three": null,
      "two": null,
      "thirteen": null,
      "eight": null,
      "fourteen": null,
      "four": null,
      "sixteen": null,
      "twelve": null,
      "eleven": null,
      "ten": null,
      "five": null
    },
    "slotToElicit": "two",
    "message": {
      "contentType": "PlainText",
      "content": "That is correct! The answer is \"2012\". New score is 100 points! For 200 points: What Amazon-favorite sport was introduced in re:Invent attendees in 2016?"
    }
  }
}
